src/Http/Controllers/PasswordResetController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers;
+
+use Blumilk\Meetup\Core\Http\Requests\PasswordResetRequest;
+use Blumilk\Meetup\Core\Http\Requests\PasswordUpdateRequest;
+use Illuminate\Auth\Events\PasswordReset;
+use Illuminate\Auth\Passwords\PasswordBrokerManager;
+use Illuminate\Contracts\Auth\PasswordBroker;
+use Illuminate\Contracts\Hashing\Hasher;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Support\Facades\Password;
+use Illuminate\Support\Str;
+use Illuminate\View\View;
+
+class PasswordResetController extends Controller
+{
+    public function __construct(
+        protected PasswordBrokerManager $passwordBrokerManager,
+        protected Hasher $hash,
+    ) {}
+
+    public function create(): View
+    {
+        return view("user.password.forgot-password");
+    }
+
+    public function store(PasswordResetRequest $request): View|RedirectResponse
+    {
+        $status = $this->passwordBrokerManager->sendResetLink(
+            $request->validated(),
+        );
+
+        return $status === PasswordBroker::RESET_LINK_SENT
+            ? \view("user.password.dashboard")->with(["status" => __($status)])
```

Comment:
```suggestion
            ? view("user.password.dashboard")->with(["status" => __($status)])
```

Why this `view()` helper is used with slash?